.github/workflows/path_detection.yml

```diff
@@ -0,0 +1,37 @@
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+name: Check for missing Dockerfile paths in repo comps
+
+on:
+  pull_request:
+    branches: [main]
+    types: [opened, reopened, ready_for_review, synchronize]
+
+jobs:
+  check-dockerfile-paths:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Clean Up Working Directory
+        run: sudo rm -rf ${{github.workspace}}/*
+
+      - name: Checkout repo GenAIExamples
+        uses: actions/checkout@v4
+
+      - name: Clone repo GenAIComps
+        run: |
+          cd ..
+          git clone https://github.com/opea-project/GenAIComps.git
+
+      - name: Check for missing Dockerfile paths in GenAIComps
+        run: |
+          cd ${{github.workspace}}
+          grep -Ern 'docker build .* -f comps/.+/Dockerfile' --include='*.md' . | while IFS=: read -r file line content; do
+              dockerfile_path=$(echo "$content" | awk -F '-f ' '{print $2}' | awk '{print $1}')
+              if [[ ! -f "../GenAIComps/${dockerfile_path}" ]]; then
+                  echo "Missing Dockerfile: ${dockerfile_path} (Referenced in ${file}:${line})"
+                  exit 1
```

Comment:
Let's finish check all the files, and raise issue at the end.

Reply:
Done.